README.md

```diff
@@ -184,6 +240,7 @@ filemap            Map files into a Flux instance
 ```
 
 Why would I want to do this? Shouldn't I define urgency when I submit?
+(Should this be an admin command not accessible to standard users?)
```

Comment:
Maybe.

Reply:
Let's bring with up with Tom when we present to the main group.

Reply:
Sounds good, resolving here.